### PR TITLE
Azure PS updates.

### PR DESCRIPTION
**Pull request by ericsciple**

### CODE REVIEW

File: Tasks/Common/AzureHelpers/PrivateFunctions.ps1 [hunk 1]
```diff
@@ -52,7 +58,7 @@ function Initialize-AzureSubscription {
         $store.Add($certificate)
         $store.Close()
         $additional = @{ }
-        if ($script:azureModuleVersion -lt ([version]'0.8.8')) {
+        if ($script:isClassic -and $script:classicVersion -lt ([version]'0.8.8')) {
```
Comment (TingluoHuang) [inline Tasks/Common/AzureHelpers/PrivateFunctions.ps1]: we will throw on 0.8.10 on import-azuremodule.

**Comment by TingluoHuang** [comment]
Looks good.

File: Tasks/Common/AzureHelpers/PrivateFunctions.ps1 (hunk 1 above)
Comment (ericsciple) [inline Tasks/Common/AzureHelpers/PrivateFunctions.ps1]: Good catch. I will fix when I add unit tests in a follow-up PR.

File: Tasks/Common/AzureHelpers/PrivateFunctions.ps1 [hunk 2]
```diff
@@ -62,107 +68,217 @@ function Initialize-AzureSubscription {
             $additional['CurrentStorageAccountName'] = $StorageAccount
         }
 
+        Write-Host "##[command]Set-AzureSubscription -SubscriptionName $($Endpoint.Data.SubscriptionName) -SubscriptionId $($Endpoint.Data.SubscriptionId) -Certificate ******** $(Format-Splat $additional)"
         Set-AzureSubscription -SubscriptionName $Endpoint.Data.SubscriptionName -SubscriptionId $Endpoint.Data.SubscriptionId -Certificate $certificate @additional
         Set-CurrentAzureSubscription -SubscriptionId $Endpoint.Data.SubscriptionId
```
Comment (ericsciple) [inline Tasks/Common/AzureHelpers/PrivateFunctions.ps1]: Per Yas, add storage account when calling Set-CurrentAzureSubscription for certificate.

**Pull request merged**